src/Caching/SqlServer/src/MonoSqlParameterCollectionExtensions.cs

```diff
@@ -22,23 +22,22 @@ internal static class MonoSqlParameterCollectionExtensions
             this SqlParameterCollection parameters,
             DateTimeOffset utcTime)
         {
-            return parameters.AddWithValue(Columns.Names.ExpiresAtTime, SqlDbType.DateTime, utcTime.UtcDateTime);
+            return parameters.AddWithValue(Columns.Names.ExpiresAtTime, SqlDbType.VarChar, utcTime.UtcDateTime.ToString());
         }
 
-
         public static SqlParameterCollection AddAbsoluteExpirationMono(
                     this SqlParameterCollection parameters,
                     DateTimeOffset? utcTime)
         {
             if (utcTime.HasValue)
             {
                 return parameters.AddWithValue(
-                    Columns.Names.AbsoluteExpiration, SqlDbType.DateTime, utcTime.Value.UtcDateTime);
+                    Columns.Names.AbsoluteExpiration, SqlDbType.VarChar, utcTime.Value.UtcDateTime.ToString());
```

Comment:
Is this changing the schema of the table?  How does that work with  existing tables that had a different schema?

Reply:
If this is a breaking schema change then that seems like a non-starter. Or is there also a migration plan proposed?